README.md

```diff
@@ -1 +1,58 @@
-# app-merchant
+# AppMerchant Package
+[![Latest Stable Version](https://poser.pugx.org/spryker/app-merchant/v/stable.svg)](https://packagist.org/packages/spryker/app-merchant)
+[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%208.1-8892BF.svg)](https://php.net/)
+
+Provides SyncAPI and AsyncAPI schema files and the needed code to be used in a Payment Service Provider App.
+
+## Installation
+
+```
+composer require spryker/app-merchant
+```
+
+### Configure
+
+#### App Identifier
+
+config/Shared/config_default.php
+
+```
+use Spryker\Shared\AppMerchant\AppConstants;
+
+$config[AppConstants::APP_IDENTIFIER] = getenv('APP_IDENTIFIER') ?: 'hello-world';
+```
+
+### Testing the AppMerchant
+
+You can test the AppMerchant as usual with Codeception. Before that you need to run some commands:
+
+```
+composer setup
+```
+
+With these commands you've set up the AppMerchant and can start the tests
+
+```
+vendor/bin/codecept build
+vendor/bin/codecept run
+```
```

Comment:
why do people need to test AppMerchant package?

Reply:
When they want to develop it. We open the platform and I guess this one will also be open.